app/src/main/java/com/google/samples/apps/sunflower/utilities/SingleLiveEvent.kt

```diff
@@ -0,0 +1,76 @@
+/*
```

Comment:
Use `EventObserver` pattern instead `SingleLiveEvent`
Check: http://tinyurl.com/y7e8hq4v
There's a blog link which describes how different between `EventObserver` & `SingleLiveEvent` which is important.

Reply:
@XinyueZ OK, I'll learn it.